js/tex.jsx

```diff
@@ -38,6 +40,19 @@ function doProcess(callback) {
     });
 }
 
+// Make content only visible to screen readers.
+// Both collegeboard.org and Bootstrap 3 use this exact implementation.
+var srOnly = {
+    border: 0,
+    clip: "rect(0,0,0,0)",
+    height: "1px",
+    margin: "-1px",
+    overflow: "hidden",
+    padding: 0,
+    position: "absolute",
+    width: "1px"
+};
```

Comment:
I thought about that but it doesn't appear as if we have any stylesheets for the react-components - and adding in one for a single stylesheet rule seems cumbersome. I'm open to thoughts here, though!